geospaas/base_viewer/forms.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.gis import forms
+from django.utils import timezone
+from datetime import tzinfo
+from leaflet.forms.widgets import LeafletWidget
+import pytz
```

Comment:
You don't need that, since you already import `timezone` from `django.utils`, you can use `timezone.utc` instead of `pytz.utc`.

Reply:
done